docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -141,44 +149,148 @@ Restrictions
+
 Vivarium Modeling Strategy
 --------------------------
 
+We will use an **exposure model** (or **prevalence-only model** or **propensity
+model**) for a vitamin A deficiency, in which each simulant is initialized with a "propensity" for vitamin A deficiency, and the simulant's vitamin A status is determined by comparing this
+propensity to the overall VAD exposure/prevalence in the population.
+Such
+propensity/exposure models have been used in Vivarium for other risk factors and
+risk-attributable causes, such as child stunting, :ref:`child wasting/PEM
+<2017_cause_pem>`, and :ref:`iron deficiency anemia
+<2017_cause_iron_deficiency>`.
+
+In more detail, the basic strategy is to initialize each simulant with a
+propensity score distributed uniformly in [0,1], then compare this propensity
+score with the (location/age/sex/year/intervention-status)-dependent prevalence
+of vitamin A deficiency at each time step to determine whether the simulant has
+VAD during that time step. Each simulant's propensity is assigned only once, but
+the underlying prevalence distribution can change throughout the course of the
+simulation, which may result in a change in the simulant's vitamin A status.
+
+In particular, our modeling strategy will **not** explicitly use incidence or
+remission data for vitamin A deficiency, but only *prevalence* (which is the
+same as the exposure data for the VAD risk factor). The rationale for this approach is twofold:
+
+1.  We want to guarantee that the simulated baseline prevalence of vitamin A
+    deficiency matches the prevalence data from GBD, which is likely more
+    trustworthy than incidence and remission data.
+
+2.  Relative risks from the literature about the effects of vitamin A
+    supplementation or fortification on vitamin A status are best interpreted as
+    risk ratios for prevalence of vitamin A deficiency. The exposure model
+    provides a way to directly model these effect sizes in a way that preserves
+    this interpretation.
+
+.. todo::
+
+  Verify that effect sizes on VAD should actually be interpreted as described
+  above, and that the prevalence-only model is a good way to accurately
+  represent these numbers.
+
+  Explain why the prevalence-only model is a reasonable strategy, citing
+  incidence, remission, and prevalence data, as well as expert opinions about
+  VAD. (Perhaps this explanation should come later, e.g. in the Assumptions and
+  Limitations section.)
+
+Following is a more detailed description of how the exposure model for VAD
+should work.
+
+Determining Vitamin A Status
+++++++++++++++++++++++++++++
+
+At each time step, Vivarium needs to determine whether each simulant has vitamin
+A deficiency. To do so, follow these steps:
+
+1.  **Initialize:** When simulant :math:`i` enters the simulation (e.g. at the
+    start of the simulation or at the time step when the simulant is born),
+    assign the simulant a random number :math:`v_i \sim
```

Comment:
I think we should standardize the notation for these things.  I follow `scipy` conventions in the code and I think they're reasonable:

x_i - the random variable (the exposure)
q_i - the percentile or propensity of the exposure x_i in the distribution (called q because the inverse cdf is the quantile function, though also called the percent point function).

Reply:
Initialize should also describe how we actually get `x_i`.

Reply:
@collijk In this case `x_i` would be a binary variable, "has VAD"/"does not have VAD", correct? How would you initialize this before following the procedure in the "Update" step? I assumed the "update" part would first happen during the same time step as initialization.

Reply:
The procedure is the same as in update, but it has to happen before a time step takes place.  The value of other attributes the simulant is initialized with (e.g. whether or not they are receiving vitamin a fortification) may be dependent on their initial vitamin a deficiency status.  This is a fencepost error (https://en.wikipedia.org/wiki/Off-by-one_error#Fencepost_error).  All attributes of a simulant must be assigned an initial value before the first time step starts or you introduce extremely hairy issues into the order you must update simulant state each time step.

Here's the procedure:

1. Initialization:
 - Sample propensity and store it for all time.
 - Use attributes initial distribution is conditional on (age, sex, year) to construct probability mass function
 - Use propensity to sample pmf and assign initial status.
2. Update:
 - Your procedure looks good here.